src/components/Button.tsx

```diff
@@ -55,6 +58,11 @@ const commonClasses = (
     boxShadow('shadow-2xl', 'hover:shadow-lg', 'active:shadow-md'),
     width(fullWidth ? 'w-full' : 'w-fit'),
     textAlign(center ? 'text-center' : undefined),
+    fontSize(small ? 'text-sm' : 'text-lg'),
+    lineHeight(small ? 'leading-5' : 'leading-7'),
+    type !== 'tertiary'
+      ? padding(small ? 'py-2' : 'py-4', small ? 'px-4' : 'px-6')
```

Comment:
I think doing the `small ? {'py-2': true, 'px-4': true} : ...` is faster than checking for `small` two times

Reply:
Idk about that. Thanks)